symfony/plugins/orangehrmPimPlugin/lib/dao/EmployeeDao.php

```diff
@@ -1474,7 +1474,7 @@ public function saveEmployeeTaxExemptions(EmpUsTaxExemption $empUsTaxExemption)
      * @param int $membershipId
      * @return array EmployeeMembership 
      */
-    public function getEmployeeMemberships($empNumber, $membershipId = null) {
+    public function getEmployeeMemberships($empNumber, $membershipId) {
```

Comment:
EmployeeDao functions have been changed. Aren't the unit tests modified to reflect this change?